### ISSUE TITLE
One-hot to label correspondence

### ISSUE THREAD

**Issue by Abello966** [bonlime/keras-deeplab-v3-plus#51]
Hello!
First of all, thanks for your repository, it's very helpful!

Dumb question, but I can't seem to find the correspondence between the label numbers and the label meanings. I'm reading the PASCAL-VOC devkit and there's a table in alphabetic order, is it the same numbering of the labels?

I'd like to produce some images similar to the ones on the README, where we have both the segmentation result and the label caption, if you could provide the code or the info i'd be very glad

**Comment by bonlime** [comment]
I used code from  [original repo](https://github.com/tensorflow/models/blob/master/research/deeplab/deeplab_demo.ipynb)  to make images in README. You need to do the same image manipulations they do in `DeepLabModel.run()`, then use keras `model.predict()` on this image, and then use `vis_segmentation` function

**Comment by yinhangbupt** [comment]
label_index_dict = { index: name for index, name in enumerate(['background', 'aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus',  'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike',  'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tv'])}
The "labels" from the code of "How to get labels" , is a matrix of label numbers, indicating label number of pixels.
So we could know the label meaning of each pixel.
